cmd/app.go

```diff
@@ -1080,6 +1083,92 @@ func createNamespace(c *cliclient.MasterClient, n string) error {
 	return nil
 }
 
+func processFileInputs(
+	ctx *cli.Context,
+	c *cliclient.MasterClient,
+	tv *managementClient.TemplateVersion,
+	answers map[string]string,
+	values string,
+	interactive bool) (map[string]string, string, error) {
+	var err error
+	if ctx.String("answers") != "" && ctx.String("values") != "" {
+		return nil, "", fmt.Errorf("cannot process an answers file and a values file")
+	}
+	if hasAnswers(ctx.String("answers"), answers) || !hasAnswers(ctx.String("answers"), answers) && !hasValues(ctx.String("values"), values) {
```

Comment:
you can remove the second `!hasAnswers(ctx.String("answers"), answers)` as the condition won't be evaluated unless that is the case